tests/test_connections/test_abci/test_abci.py

```diff
@@ -462,6 +471,9 @@ async def test_connection_standalone_tendermint_setup() -> None:
     await asyncio.sleep(2.0)
     await connection.disconnect()
 
+    del os.environ["LOG_FILE"]
+    temp_dir.cleanup()
```

Comment:
this needs to be in a `finally` block; otherwise it's not guaranteed we clean up